include/nuttx/power/pm.h

```diff
@@ -631,13 +801,27 @@ void pm_auto_updatestate(int domain);
  * avoid so much conditional compilation in driver code when PM is disabled:
  */
 
+#  define PM_WAKELOCK_INITIALIZER(n,d,s)      {0}
```

Comment:
is `PM_WAKELOCK_INITIALIZER` going to be used away from `PM_WAKELOCK_DECLARE` or `PM_WAKELOCK_DECLARE_STATIC`?

Reply:
If not, then empty initialiser `#  define PM_WAKELOCK_INITIALIZER(n,d,s)      {0}` can be remove